### ISSUE TITLE
Notifications shaking when `IModelApp.notifications.outputMessage` is used

### ISSUE THREAD

**Issue by ImVeryLost** [iTwin/itwinjs-core#3564]
**Describe the bug**
When multiple notifications are opened using `IModelApp.notifications.outputMessage`, and the top one is closed, the remaining ones visually shake for a second.

**To Reproduce**

1. Go to [Add toolbar button sample](https://www.itwinjs.org/sandboxes/iTwinPlatform/Add%20Toolbar%20Button)
2. In the ToolbarButtonProvider displayInfoToast function, edit the content so it displays the toast using outputMessage and the toast type is of "OutputMessageType.Sticky" type:
``` 
   const toast = new NotifyMessageDetails(OutputMessagePriority.Error, message);
    toast.msgType =  OutputMessageType.Sticky;
    toast.openAlert = OutputMessageAlert.Balloon;
    IModelApp.notifications.outputMessage(toast);
```

4. Run the sample, click the "lightbulb" a few times so at least 2 toasts are displayed.
5. Close the **top** one (order is important).
6. The SVGs (exclamation mark and the close icon) in the bottom toast message are visually shaky for a few seconds.

Note that behaviour was observed elsewhere as well, so its not sample/sandbox specific.

**Expected behavior**
The SVGs should not shake.

**Screenshots**
https://user-images.githubusercontent.com/61872067/165777882-6676e73b-eb94-4f28-9a59-89bdedceba0c.mp4

**Desktop (please complete the applicable information):**

- Browser - seen in firefox, 99.0.1. likely applicable in chrome as well.
- iTwin.js Version - 3.2.x, and whatever the itwin.js samples are using.

**Comment by NancyMcCallB** [comment]
We have seen this problem -- it appears to have something to do with the center justification of the messages. We have a fix in the works, but it will probably be in the 3.3 release. I'll attach this issue to our internal tracking. Thanks!

**Comment by aruniverse** [comment]
@ImVeryLost this was fixed in 3.3, please try upgrading to that!

**Comment by ImVeryLost** [comment]
> @ImVeryLost this was fixed in 3.3, please try upgrading to that!

Seems fine now, thanks!
